schemas/styles/csl-choose.rnc

```diff
@@ -19,7 +19,7 @@ div {
   
   ## This element allows for more complex conditional logic; for 
   ## example, if you need to specify mixed "match" rules.
-  condition.elem = element cs:condition { match?, condition.atts }
+  condition.elem = element cs:condition { match, condition.atts }
```

Comment:
I mean, can do whatever you guys think best, and I'll defer to you, since you deal more with style authoring and editing than I ever have. 

I was just hoping to make the rules clear in the validation specification, and thinking that this wouldn't require the element:

``` xml
<choose>
  <if>
    <condition type="book"/>
    <condition variable="title"/>
  </if>
</choose>
```

... which would just be:

``` xml
<choose>
  <if type="book" variable="title">
    ...
  </if>
</choose>
```

But there may not be value in restricting.

Reply:
Well, but this

> ```
> <choose>
>   <if>
>     <condition type="book chapter" match="none"/>
>     <condition variable="title"/>
>   </if>
> </choose>
> ```

Can't be transformed into the current syntax without adding nested `choose` elements.

Reply:
Correct, because the match conditions differ.

I wonder if we made a mistake allowing a default match value.

Reply:
I'll revert this commit.

Reply:
No, leave it. To be clear, I **do think** that the default should be `match="all"` for `if`, `else-if`, and `condition`. I just am not so familiar with the rnc syntax to know if that's the case here.

Reply:
RelaxNG, by design, has no notion of default values and such. That's only something specified in documentation, etc.

What I changed was requiring `@match`.

I think we'll only know for sure on these details with testing.

Reply:
Okay great, then do leave it as optional